lib/api/src/modules/stories.ts

```diff
@@ -1,4 +1,5 @@
-import { toId, sanitize } from '@storybook/router/dist/utils';
+// FIXME: we shouldn't import from dist but there are no types otherwise
+import { toId, sanitize, parseKind } from '@storybook/router/dist/utils';
```

Comment:
@shilman you can do:
`import { toId, sanitize, parseKind } from '@storybook/router';`

because `index.ts` of `@storybook/router` is exporting everything from `/utils` folder:
```ts
export * from './utils';
export * from './router';
```